migrations/20220607041349_add_mfa_schema.up.sql

```diff
@@ -0,0 +1,47 @@
+-- See: https://stackoverflow.com/questions/7624919/check-if-a-user-defined-type-already-exists-in-postgresql/48382296#48382296
+DO $$ BEGIN
+    CREATE TYPE factor_type AS ENUM('phone', 'webauthn');
```

Comment:
can we just solve this by adding a namespace and prefix the factor type with `mfa-` ?

Reply:
Hmm do you mean like:

```sql
CREATE TYPE mfa.factor_type AS ENUM('mfa-phone', 'mfa-webauthn');
```

Reply:
yup, or even `phone-factor` , `webauthn-factor` whichever you prefer

Reply:
hmm how would (2) occur, the `schema_migrations` table wouldn't be dropped by the migration tool unless the user does it explicitly right?

Reply:
Yep the main use case I had in mind was a support ticket with migration issues